bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -117,6 +118,38 @@ def add_hofers(filename, data):
         )
 
 
+def parse_cve_id(cve_id):
+    cve_year, cve_order = cve_id.split('-')[1:]
+    return int(cve_year), int(cve_order)
+
+
+def add_or_update_cve(data):
```

Comment:
Do you think it's worth adding tests for these two methods? Might be nice just to have an example of expected input data format.